### PR TITLE
(feature) add LINE notify to notifier

### PR DESCRIPTION
**Pull request by huydx**
## add LINE to notifier

LINE is very famous chat message at many asia countries (japan/taiwan/thai...). Adding notification to LINE will help user in those countries to access information better. The implement come from a service from LINE called LINE notify (which implement light-weight notification method): https://notify-bot.line.me/en/

About LINE: http://line.me/
How it looks:

### CODE REVIEW

**Comment by huydx** [comment]
@bergquist I fixed using latest upstream, can you take a look :)

File: pkg/services/notifications/webhook.go [hunk 1]
```diff
@@ -63,6 +64,12 @@ func sendWebRequestSync(ctx context.Context, webhook *Webhook) error {
 		request.Header.Add("Authorization", util.GetBasicAuthHeader(webhook.User, webhook.Password))
 	}
 
+	if len(webhook.HttpHeader) != 0 {
```
Comment (bergquist) [inline pkg/services/notifications/webhook.go]: This if statement can be removed. If webhook.HttpHeader is empty the loop wont do anything

https://play.golang.org/p/Wc_ajoYUfh

**Review by bergquist — CHANGES_REQUESTED** [review]

File: pkg/services/notifications/webhook.go (hunk 1 above)
Comment (bergquist) [inline pkg/services/notifications/webhook.go]: Will merge once the if statement is removed.

**Review by bergquist — APPROVED** [review]

File: pkg/services/notifications/webhook.go (hunk 1 above)
Comment (bergquist) [inline pkg/services/notifications/webhook.go]: Fixed it myself and merged. Awesome work!

**Comment by huydx** [comment]
thanks @bergquist a lot!!!

**Pull request merged**